include/z64player.h

```diff
@@ -688,11 +688,11 @@ typedef struct Player {
     /* 0x06C0 */ s16        unk_6C0;
     /* 0x06C2 */ s16        unk_6C2;
     /* 0x06C4 */ f32        unk_6C4;
-    /* 0x06C8 */ SkelAnime  skelAnime2;
-    /* 0x070C */ Vec3s      jointTable2[PLAYER_LIMB_BUF_COUNT];
-    /* 0x079C */ Vec3s      morphTable2[PLAYER_LIMB_BUF_COUNT];
-    /* 0x082C */ PlayerFunc82C func_82C;
-    /* 0x0830 */ f32        unk_830;
+    /* 0x06C8 */ SkelAnime  skelAnimeUpper;
+    /* 0x070C */ Vec3s      jointTableUpper[PLAYER_LIMB_BUF_COUNT];
+    /* 0x079C */ Vec3s      morphTableUpper[PLAYER_LIMB_BUF_COUNT];
```

Comment:
Stalfos (test.c) has 
```c
/* 0x4A8 */ SkelAnime upperSkelanime;
/* 0x4EC */ Vec3s upperJointTable[STALFOS_LIMB_MAX];
/* 0x65A */ Vec3s upperMorphTable[STALFOS_LIMB_MAX];
```
Which puts the upper before the names instead of after, it should probably be consistent (also noting Skelanime -> SkelAnime in test.c but can be done in a later cleanup)

Reply:
I went with this mainly because I want to see `skelAnime` right away in the name when its passed around to various animation functions. Do agree stalfos should be consistent but would rather that be done in a separate stalfos pass